Description: use krb5-config to query the necessary library options
Origin: vendor
Bug-Debian: http://bugs.debian.org/622090
Forwarded: no
Author: Steve Langasek <steve.langasek@canonical.com>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-04-10

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -18,22 +18,10 @@
 
 ##### DO NOT CHANGE ANYTHING BELOW HERE #####
 
-# check for libk5crypto -- only in krb5-1.1 and above
-print "Checking for libk5crypto...";
-my $cryptolib;
-if ( -r "${KRB5_LIBDIR}/libk5crypto.a" || -r "${KRB5_LIBDIR}/libk5crypto.so" ) {
-	print "yes\n";
-	$cryptolib = '-lk5crypto';
-}
-else {
-	print "no.  I'll use libcrypto instead.\n";
-	$cryptolib = '-lcrypto';
-}
-
 WriteMakefile(
     'NAME'	=> 'Authen::Krb5',
     'VERSION_FROM' => 'Krb5.pm',
-    'LIBS'	=> ["-L${KRB5_LIBDIR} -lkrb5 ${cryptolib} -lcom_err $KRB5_EXTRALIBS"],
+    'LIBS'     => [`krb5-config --libs`],
     'DEFINE'	=> '',
-    'INC'	=> "-I${KRB5_INCDIR} $KRB5_EXTRAINCS"
+    'INC'      => `krb5-config --cflags`
 );
